gradio/helpers.py

```diff
@@ -103,7 +103,7 @@ def __init__(
         run_on_click: bool = False,
         preprocess: bool = True,
         postprocess: bool = True,
-        api_name: str | None | Literal[False] = False,
+        api_name: str | Literal[False] = "load_example",
```

Comment:
There's no reason to support `api_name` here since `None` will default to "load_example" in all cases